README.md

```diff
@@ -20,53 +26,122 @@ to avoid command conflicts with other bots in the server, users can customize UH
 lines. The customizability ensures a seamless and non-disruptive experience across various 
 functionalities.
 
+## Installation Guide
+
+### Before You Start: Setting Up a Discord Server.
+
+In order to install UHelper, you have to create your own server or are already an administrator in 
+an existing server. If you have not used Discord before or do not know how to work with servers, it 
+is recommended that you read through [this beginner
+'s guide](https://support.discord.com/hc/en-us/articles/360045138571-Beginner-s-Guide-to-Discord#h_efc9b7bc-47bc-4212-8b9c-c0fa76573cfe).
+
+### How to Invite UHelper to Your Server
+
+![png](assets/0.png)
+
+If you can see a screen similar as above, that means you've successfully created your own server. 
+You are now ready to invite UHelper to help you through your journey as a student. Next, click this 
+[link](https://discord.com/oauth2/authorize?client_id=1253813199397191863&permissions=1126727225142272&integration_type=0&scope=bot). The link will take you to a webpage that not only informs you that you are about to 
+UHelper to a server, but also prompts you to select which server you want UHelper to join in. 
+
+![png](assets/1.png)
+
+Here, you can select the server you want (it's under the Add to Server section!) and afterward, 
+press continue.
+
+![png](assets/2.png)
+
+Then, you will be asked to confirm that you wish to grant some permissions to UHelper. Because these 
+are all essential for the bot to function, you should not uncheck any one of them. Scroll
+down to the bottom to find the "authorize" button and click it.
+
+![png](assets/3.png)
+
+Seeing this screen means that UHelper has successfully joined your server. Congratulations! The next
+part of this guide will teach you how to run UHelper.
 
-## Project Progresses 👀
+### How to Run UHelper in a Server
+
+![png](assets/4.png)
+
+UHelper joined the server, but it cannot listen to your commands as it is not online. Open up 
+IntelliJ IDEA and go to ca.unknown.bot.app.Main. Go to Edit Configurations to add a new configuration
+setting for this main class.
+
+![png](assets/5.png)
+
+In the edit configuration window, choose Java 11 or 17 and Project.Main under the Build and Run section.
+For the main class box, type ca.unknown.bot.app.Main. Next, importantly, you should add a bot 
+token as an environmental variable... but where is the token, you ask? Discord enforces 
+strict rules to its bot developers and one of them is that you can't post
+any bot token online due to security reasons. If you do, it will immediately detect the token and 
+disable it. But, you can still have access to the UHelper's token! We've attached the token via the 
+comment section of the Phase 1 submission.
+
+Once you find the token, set it up as an environmental variable. Click the apply button to save 
```

Comment:
This is incredible work, thank you for typing it up so clearly! Maybe it could help to say that he needs to type TOKEN= ... because I also had trouble adding my bot to the server because I didn't know how to do that in the env variable section until Cindy sent me a screenshot of how it should look.

Reply:
Thank you so much for reading through all this! I will make sure to add that he needs to explicitly type TOKEN= in the installation guide.